Dockerfile

```diff
@@ -51,7 +53,8 @@ COPY makefile.manifest.inc ${SRC_ROOT}
 
 # Build
 RUN cd ${SRC_ROOT} && \
-    make SYSFLOW_VERSION=$VERSION \
+    make BACKEND_TAG=otel \
```

Comment:
The default backend shouldn't be `otel`, otherwise, our image won't support the default `flatrecord` anymore. 
We should probably build the two backends, and allow the backend to be selectable in the pipeline configuration. 
cc: @terylt thoughts?

Reply:
Yes, as long as it's not that big.. we can compile both into the processor... and make it selectable.